tools/linker/ObjCExtensions.cs

```diff
@@ -28,7 +28,36 @@ internal static void Initialize ()
 			ObjCRuntime = profile.GetNamespace ("ObjCRuntime");
 			Security = profile.GetNamespace ("Security");
 			StoreKit = profile.GetNamespace ("StoreKit");
+			GameController = profile.GetNamespace ("GameController");
```

Comment:
Please verify this looks sane for iOS. I spot checked src/build/ios/Constants.cs where I wasn't sure.